src/sentry/static/sentry/app/components/panels/panelItem.jsx

```diff
@@ -1,18 +1,25 @@
 import {Flex} from 'grid-emotion';
 import PropTypes from 'prop-types';
 import styled from 'react-emotion';
+import isPropValid from '@emotion/is-prop-valid';
 
-const PanelItem = styled(Flex)`
-  border-bottom: 1px solid ${p => p.theme.borderLight};
+const getBorders = p => `
+  border-bottom:  1px solid ${p.theme.borderLight};
 
   &:last-child {
-    border: 0;
+    border-bottom: 0;
   }
 `;
 
+const PanelItem = styled(Flex, {shouldForwardProp: isPropValid})`
+  ${p => !p.borderless && getBorders(p)};
```

Comment:
solid syntax highlighting here

Reply:
you could do
```suggestion
  ${getBorders};
```

And handle it in `getBorders`